[Bug]: terraform build failed

### Problem description

go: downloading golang.org/x/exp v0.0.0-20200224162631-6cc2880d07d6
Downloading https://github.com/lib/pq/commit/6a102c04ac8dc082f1684b0488275575c374cb4c.patch
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1815  100  1815    0     0   6817      0 --:--:-- --:--:-- --:--:--  6823
# github.com/lib/pq
../../../../pkg/mod/github.com/lib/pq@v1.10.3/user_posix.go:13:6: userCurrent redeclared in this block
        ../../../../pkg/mod/github.com/lib/pq@v1.10.3/user_other.go:8:6: other declaration of userCurrent
builder@1032af077e44:~/termux-packages$
[terraform.txt](https://github.com/termux/termux-packages/files/9468535/terraform.txt)


### What steps will reproduce the bug?

build

### What is the expected behavior?

_No response_

### System information

termux-info:

```

```

--- a/packages/terraform/build.sh
+++ b/packages/terraform/build.sh
@@ -29,6 +29,10 @@ termux_step_make() {
 		chmod 0755 "$(dirname "$f")"
 		chmod 0644 "${f}"
 		patch --silent -p1 -d "$(dirname "$f")" <"${TERMUX_PKG_TMPDIR}"/patch1
+		# The patch above does not fix build issue for some reason.
+		# Alternative workaround:
+		rm -f "${f}"
+		echo "package pq" > "${f}"
 	done
 
 	local GO_LDFLAGS="-X 'github.com/hashicorp/terraform/version.Prerelease='"

